src/test/Fake.DotNet.Cli.IntegrationTests/TemplateTests.fs

```diff
@@ -0,0 +1,91 @@
+﻿module Fake.DotNet.Cli.IntegrationTests.TemplateTests
+
+open Expecto
+open System
+open System.IO
+
+open Fake.Core
+open Fake.DotNet
+open Fake.IO
+open Fake.IO.Globbing
+
+let templateProj = "fake-template.fsproj"
+let templatePackageName = "fake-template"
+let templateName = "fake"
+
+//TODO: add DotNetCli helpers for the `new` command
+
+let uninstallTemplate () =
+    DotNet.exec id "new" (sprintf "-u %s" templatePackageName)
+
+let installTemplateFrom pathToNupkg =
+    DotNet.exec id "new" (sprintf "-i %s" pathToNupkg)
+
+type BootstrapKind =
+| Tool
+| Project
+| None
+with override x.ToString () = match x with | Tool -> "tool" | Project -> "project" | None -> "none"
+
+let shouldSucceed message (r: ProcessResult) =
+    Expect.isTrue r.OK (sprintf "%s. Results:\n:%A" message r)
+
+let runTemplate rootDir kind = async {
+    Directory.ensure rootDir
+    let dotnet = DotNet.Options.Create().DotNetCliPath
+    let proc = 
+        Process.getProc (fun p ->
+            p.WithWorkingDirectory(rootDir)
+             .WithFileName(dotnet)
+             .WithArguments(sprintf "new %s --bootstrap %s" templateName (string kind))
+             .WithRedirectStandardInput(true)
+             .WithRedirectStandardOutput(true)
+             .WithRedirectStandardError(true)
+        )
+    
+    proc.Start () |> ignore
+    do! Async.Sleep 2000
```

Comment:
Do we really need that?

Reply:
Yeah, I was guessing at how long to wait after invoking the template gen, so that the prompt would be active at the point that I sent the Y character.

Reply:
Did you try `--allow-scripts yes` (not the most important - but I guess it would not introduce a potential race condition)

Reply:
No, I'll give that a try when I add the new tests